test/tools/ossfuzz/abiV2ProtoFuzzer.cpp

```diff
@@ -38,7 +38,7 @@ static size_t const expectedOutputLength = 32;
 /// Expected output value is decimal 1000 or hex 03E8
```

Comment:
The change below suggests the comment needs to be updated.

Reply:
Oh right! Thank you @ekpyron , pushed a fix for this.